### DISCUSSION TITLE
Differences between image files

### DISCUSSION THREAD

**Discussion by starbucks2010** [cfig/Android_boot_image_editor#98] [Q&A] [answered]
Hello,

Thank you for keeping updated this beautiful tool.
I have a question tho: what is the difference between the 4 .img files created (.img.clear, .img.google, .img.signed and .img.signed2) and which one is most suitable to be used?

Thanks.

**Comment by cfig** [comment, marked as answer]
Hi,
Thanks for using this tool.

.img.clear,  and .img.google are identical. They are the original boot image.
.img.signed  and .img.signed2 are identical. They have additional signatures compared with ".img.clear".

For most cases, ".img.signed" should be the choice to use.
Cheers~
